### PR TITLE
chore(set-up-react-app)-181414707

### PR DESCRIPTION
**Pull request by Robertishimwe**
### 1. Name of task
Setup an empty react project
### 2. Pivot Tracker ID
[181414707](https://www.pivotaltracker.com/story/show/181414707)
### 3. Branch Name
ch-set-up-react-app-181414707
### 4. Description of task
should create a react project from scratch using Webpack 4, Babel, and the Airbnb linting option.

### CODE REVIEW

**Review by itisWasp — CHANGES_REQUESTED** [review]
Improve the commit message and make it descriptive.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.workingDirectories": [{ "mode": "auto" }]
+}
```
Comment (ericus123) [inline .vscode/settings.json]: remove this

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Robertishimwe) [reply .vscode/settings.json]: done

**Comment by Robertishimwe** [comment]
> Improve the commit message and make it descriptive.

done

**Review by itisWasp — APPROVED** [review]

**Comment by Robertishimwe** [comment]
@ericus123 is there any other thing remaining for this task?

**Review by ericus123 — APPROVED** [review]

**Pull request merged**